### ISSUE TITLE
What knockknock are you using for the recon?

### ISSUE THREAD

**Issue by thezakman** [geeknik/recon.sh#1]
What version of knockknock are you using for the recon, perhaps https://github.com/harleo/knockknock?

```knockknock $TARGET -o $OUTDIR/subdomains_knockknock.txt```

**Comment by geeknik** [comment]
Yes, that's the one. I got busy and didn't upload a README for this project yet. \m/
